uuid.go

```diff
@@ -153,6 +156,28 @@ func (u *UUID) SetVariant(v byte) {
 	}
 }
 
+// UnixNanoOf returns the number of nanoseconds in a V1 UUID
```

Comment:
Just to be very clear to consumers, would you mind adding a quip about it being the number of nanoseconds since the UNIX epoch? I definitely don't want to nitpick copy, but I'm thinking something like:

>// UnixNanoOf returns the number of nanoseconds elapsed since January 1, 1970 UTC in a V1 UUID

It might also be good to indicate that an error is returned for a non-V1 UUID here too (like we did with `TimeOf`).

Finally, please add punctuation to the sentence just so that the documentation reads better.